moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -166,15 +166,8 @@ void TrajectoryExecutionManager::initialize()
 
   // other configuration steps
   reloadControllerInformation();
-  // The default callback group for rclcpp::Node is MutuallyExclusive which means we cannot call
```

Comment:
It looks like you are overwriting this commit:
https://github.com/ros-planning/moveit2/commit/bf6fc9afb86e9d1d795007e822af97ff63432721